starlette_admin/translations/de/LC_MESSAGES/admin.po

```diff
@@ -0,0 +1,191 @@
+
+#: starlette_admin/views.py:476
+msgid "View"
+msgstr "Ansehen"
+
+#: starlette_admin/templates/edit.html:12 starlette_admin/views.py:488
+msgid "Edit"
+msgstr "Bearbeiten"
+
+#: starlette_admin/templates/modals/delete.html:15 starlette_admin/views.py:499
+msgid "Are you sure you want to delete this item?"
+msgstr "Sind Sie sicher, dass Sie dieses Element löschen möchten?"
+
+#: starlette_admin/views.py:818
+msgid "All"
+msgstr "Alle"
+
+#: starlette_admin/templates/actions.html:6
+msgid "With selected"
+msgstr "Mit ausgewählten"
+
+#: starlette_admin/templates/create.html:12
+msgid "Create"
+msgstr "Erstellen"
+
+#: starlette_admin/templates/create.html:25
+#: starlette_admin/templates/list.html:34
+#, python-format
+msgid "New %(name)s"
+msgstr "%(name)s erstellen"
+
+#: starlette_admin/templates/create.html:44
+#: starlette_admin/templates/edit.html:46
+#: starlette_admin/templates/modals/actions.html:10
+#: starlette_admin/templates/modals/delete.html:21
+msgid "Cancel"
+msgstr "Abbrechen"
+
+#: starlette_admin/templates/create.html:46
+#: starlette_admin/templates/edit.html:48
+msgid "Save and add another"
+msgstr "Speichern und weiteres hinzufügen"
+
+#: starlette_admin/templates/create.html:48
+#: starlette_admin/templates/edit.html:50
+msgid "Save and continue editing"
+msgstr "Speichern und Bearbeitung fortsetzen"
+
+#: starlette_admin/templates/create.html:49
+#: starlette_admin/templates/edit.html:51
+msgid "Save"
+msgstr "Speichern"
+
+#: starlette_admin/templates/detail.html:13
+msgid "Detail"
+msgstr "Detail"
+
+#: starlette_admin/templates/detail.html:47
+msgid "Attribute"
+msgstr "Attribut"
+
+#: starlette_admin/templates/detail.html:48
+msgid "Value"
+msgstr "Wert"
+
+#: starlette_admin/templates/edit.html:27
+#, python-format
+msgid "Edit %(name)s"
+msgstr "%(name)s bearbeiten"
+
+#: starlette_admin/templates/error.html:7
+msgid "Oops… You just found an error page"
+msgstr "Ups… Sie haben gerade eine Fehlerseite gefunden"
+
+#: starlette_admin/templates/error.html:29
+msgid "Take me home"
+msgstr "Zur Startseite"
+
+#: starlette_admin/templates/layout.html:95
+#: starlette_admin/templates/layout.html:118
+#: starlette_admin/templates/layout.html:204
+msgid "Logout"
+msgstr "Abmelden"
+
+#: starlette_admin/templates/list.html:67
+msgid "Search"
+msgstr "Suchen"
+
+#: starlette_admin/templates/login.html:16
+msgid "Login to your account"
+msgstr "Melden Sie sich bei Ihrem Konto an"
+
+#: starlette_admin/templates/login.html:32
+msgid "Username"
+msgstr "Benutzername"
+
+#: starlette_admin/templates/login.html:43
+msgid "Password"
+msgstr "Passwort"
+
+#: starlette_admin/templates/login.html:56
+msgid "Remember me"
+msgstr "Angemeldet bleiben"
+
+#: starlette_admin/templates/login.html:60
+msgid "Sign in"
+msgstr "Anmelden"
+
+#: starlette_admin/templates/row-actions.html:38
+msgid "Actions"
+msgstr "Aktionen"
+
+#: starlette_admin/templates/forms/_delete.html:13
+msgid "Checked this to delete current content of this field"
```

Comment:
typo in the English original?

```suggestion
msgid "Check this to delete current content of this field"
```

Reply:
yes! What about another PR for this since we need to update it in several places

Reply:
#526